File: JAVA/Lanche.java
/**
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package JAVA;

import java.util.Scanner;

public class Lanche {

    public static void main(String[] args) {
		Scanner sc = new Scanner( System.in );
        int cod = sc.nextInt();
        int qtd = sc.nextInt();
        sc.close();
        
        System.out.printf("Total: R$ %.2f\n", totalToPay(cod, qtd));
	  }

    /**
     * Sum total to pay.
     */
    public static float totalToPay(int cod, int qtd) {
        float result = 0;

        switch (cod) {
            case 1:
                result = qtd * (float) 4.00;
                break;
            case 2:
                result = qtd * (float) 4.50;
                break;
            case 3:
                result = qtd * (float) 5.00;
                break;
            case 4:
                result = qtd * (float) 2.00;
                break;
            case 5:
                result = qtd * (float) 1.50;
                break;
        }

        return result;
    }
}

File: GO/Lanche.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

/**
 * Sum total to pay.
 */
func totalToPay(cod int, qtd int) float32 {
	var result float32

	switch cod {
	case 1:
		result = float32(qtd) * 4.00
	case 2:
		result = float32(qtd) * 4.50
	case 3:
		result = float32(qtd) * 5.00
	case 4:
		result = float32(qtd) * 2.00
	case 5:
		result = float32(qtd) * 1.50
	}

	return result
}
